sdk/program/src/epoch_schedule.rs

```diff
@@ -28,6 +28,12 @@ pub const MAX_LEADER_SCHEDULE_EPOCH_OFFSET: u64 = 3;
 /// Based on `MAX_LOCKOUT_HISTORY` from `vote_program`.
 pub const MINIMUM_SLOTS_PER_EPOCH: u64 = 32;
 
+/// The default first normal slot on the epoch schedule, given warmup
+pub const DEFAULT_FIRST_NORMAL_SLOT: u64 = first_normal_epoch_and_slot(DEFAULT_SLOTS_PER_EPOCH).1;
```

Comment:
I don't love this new public const. Warm-up epochs are really just for tests. Fun fact: mainnet was launched with no warmup epochs.

It looks like changes in this file could be avoided entirely if 
`EpochSchedule::without_warmup()` was used in `test_epoch_half_check()`

Reply:
Ah ok, in that case I can totally remove it and update the test. I've had annoyances in the past when writing time-related tests, usually for stake pools, so I wanted to expose the default first normal slot.

On the flipside, I want a static_assert there so I don't have to run a little test to remember what the slot number is every time, but I can do it in a separate PR.